js/schematic.js

```diff
@@ -79,19 +79,20 @@
       ;
 
       // Initialize the navigation state.
-      getQueryVariables(); // populate the query variable
+      queryFromURL(); // populate the query variable
       console.log(query)
       setupTabNavigation();
       setupSignals();
 
-
       // Initialize the selected year to the most recent.
       signal.call("selectYear", null, d3.select("#chooser-year").node().value);
 
-      d3.select("a[href='#" + query.section + "']")
-          .node()
-        .click()
-      ;
+      // If the query section doesn't have a valid link, default to the first tab
+      var sec = d3.select("a[href='#" + query.section + "']");
```

Comment:
Nice, there we go, I spoke too soon. The case of no params in URL is handled here, for section at least.